PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -784,6 +1289,64 @@ HTREEITEM ProjectPanel::addFolder(HTREEITEM hTreeItem, const TCHAR *folderName)
 	return addedItem;
 }
 
+HTREEITEM ProjectPanel::addDirectory(HTREEITEM hTreeItem, const TCHAR *folderName, bool root, const TCHAR *monitorPath, const std::vector<generic_string>* filters)

```

Comment:
I am not sure... agreed, that this change would be good, but `TCHAR*` `const TCHAR*` is used all over the source. 
Imho it probably should be better consistent currently, and refactored in a separate commit for all occurrences?